tripal_chado/src/Plugin/ChadoBuddy/ChadoCvtermBuddy.php

```diff
@@ -254,8 +266,9 @@ public function getCvterm(array $conditions, array $options = []) {
    *     - db.urlprefix
    *     - db.url
    *     - buddy_record = a ChadoBuddyRecord can be used
-   *       in place of or in addition to other keys
-   * @param array $options (Optional)
+   *       in place of or in addition to other keys.
+   * @param array $options
```

Comment:
See comment above regarding options. Let's clean this up in all cases.

Reply:
Fixed in commit https://github.com/tripal/tripal/pull/2348/commits/0a1dc5b41d3abb470d22cbbdb713a8613e9898e6 as well as the exceptions